### PR TITLE
 Release 1.6.0

### PR DESCRIPTION
**Pull request by pleshakov**
### Proposed changes

Release 1.6.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,88 @@
...
+* [736](https://github.com/nginxinc/kubernetes-ingress/pull/736): Init Ingress labeled metrics on start.
+* [686](https://github.com/nginxinc/kubernetes-ingress/pull/686): Check if config map created for leader-election.
+* [664](https://github.com/nginxinc/kubernetes-ingress/pull/664): Fix reporting events for Ingress minions.
+* [632](https://github.com/nginxinc/kubernetes-ingress/pull/632): Fix hsts support when not using SSL. Thanks to [Martín Fernández](https://github.com/bilby91).
+
+HELM CHART:
+* The version of the helm chart is now 0.4.0.
+* Add new parameters to the Chart: `controller.healthCheckURI`, `controller.resources`, `controller.logLevel`, `controller.customPorts`, `controller.service.customPorts`. Added in [750](https://github.com/nginxinc/kubernetes-ingress/pull/750), [636](https://github.com/nginxinc/kubernetes-ingress/pull/636/files) thanks to [Guilherme Oki](https://github.com/guilhermeoki), [600](https://github.com/nginxinc/kubernetes-ingress/pull/600), [581](https://github.com/nginxinc/kubernetes-ingress/pull/581) thanks to [Alex Meijer](https://github.com/ameijer-corsha).
```
Comment (Dean-Coakley) [inline CHANGELOG.md]:
```suggestion
* Add new parameters to the Chart: `controller.healthCheckURI`, `controller.resources`, `controller.logLevel`, `controller.customPorts`, `controller.service.customPorts`. Added in [750](https://github.com/nginxinc/kubernetes-ingress/pull/750), [636](https://github.com/nginxinc/kubernetes-ingress/pull/636) thanks to [Guilherme Oki](https://github.com/guilhermeoki), [600](https://github.com/nginxinc/kubernetes-ingress/pull/600), [581](https://github.com/nginxinc/kubernetes-ingress/pull/581) thanks to [Alex Meijer](https://github.com/ameijer-corsha).
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,88 @@
 # Changelog
 
+### 1.6.0
+
+OVERVIEW:
+
+Release 1.6.0 includes: 
+* Improvements to VirtualServer and VirtualServerRoute resources, adding support for richer load balancing behavior, more sophisticated request routing, redirects, direct responses, and blue-green and circuit breaker patterns. The VirtualServer and VirtualServerRoute resources are enabled by default and are ready for production use;
```
Comment (Rulox) [inline CHANGELOG.md]:
```suggestion
* Improvements to VirtualServer and VirtualServerRoute resources, adding support for richer load balancing behavior, more sophisticated request routing, redirects, direct responses, and blue-green and circuit breaker patterns. The VirtualServer and VirtualServerRoute resources are enabled by default and are ready for production use.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,88 @@
...
+* Support for OpenTracing, helping you to monitor and debug complex transactions;
```
Comment (Rulox) [inline CHANGELOG.md]:
```suggestion
* Support for OpenTracing, helping you to monitor and debug complex transactions.
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,88 @@
...
+* [621](https://github.com/nginxinc/kubernetes-ingress/pull/621): Add TLS support for vs/vsr upstreams.
+* [617](https://github.com/nginxinc/kubernetes-ingress/pull/617): Add keepalive support to vs/vsr.
+* [612](https://github.com/nginxinc/kubernetes-ingress/pull/612): Add timeouts support to vs/vsr.
+* [607](https://github.com/nginxinc/kubernetes-ingress/pull/607): Add fail-timeout and max-fails support to vs/vsr.
+* [596](https://github.com/nginxinc/kubernetes-ingress/pull/596): Add lb-method support in vs and vsr.
+
+FEATURES:
+* [750](https://github.com/nginxinc/kubernetes-ingress/pull/750): Add support for health status uri customisation. 
+* [691](https://github.com/nginxinc/kubernetes-ingress/pull/691): Helper Functions for custom annotations.
+* [631](https://github.com/nginxinc/kubernetes-ingress/pull/631): Add max_conns support for nginx plus.
```
Comment (Rulox) [inline CHANGELOG.md]:
```suggestion
* [631](https://github.com/nginxinc/kubernetes-ingress/pull/631): Add max_conns support for NGINX plus.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,88 @@
...
+* [722](https://github.com/nginxinc/kubernetes-ingress/pull/722): Fix trailing leader election cm when using helm.
+* [573](https://github.com/nginxinc/kubernetes-ingress/pull/573): Use Controller name value for app selectors.
+
+CHANGES:
+* Update NGINX versions to 1.17.6.
+* Update NGINX Plus version to R20.
+* [799](https://github.com/nginxinc/kubernetes-ingress/pull/779): Enable CRDs by default. VirtualServer and VirtualServerRoute resources are now enabled by default.
+* [772](https://github.com/nginxinc/kubernetes-ingress/pull/772): Update VS/VSR version from v1alpha1 to v1. Make sure to update the `apiVersion` of your VirtualServer and VirtualServerRoute resources.
```
Comment (Rulox) [inline CHANGELOG.md]: Does it make sense to add an `UPGRADE` section explaining this? Similar of what we did for helm upgrade here: https://github.com/nginxinc/kubernetes-ingress/releases/tag/v1.5.1

**Pull request merged**